odoo/tests/common.py

```diff
@@ -715,6 +719,10 @@ def profile(self, **kwargs):
             profile_session=self.profile_session,
             **kwargs)
 
+    def _find_csrf_token(self, text):
+        csrf_token_re = re.compile("(input.+csrf_token.+value=\")([a-f0-9]{40}o[0-9]*)", re.MULTILINE)
+        return csrf_token_re.search(text).groups()[1]
```

Comment:
Since this is moved, it should be folded into a single `re.search`, compiling to a local variable is useless (either that or the `csrf_token_re` should be moved to a global)